libkbfs/block_retrieval_queue.go

```diff
@@ -163,8 +169,84 @@ func (brq *blockRetrievalQueue) notifyWorker() {
 	}
 }
 
+func (brq *blockRetrievalQueue) CacheAndPrefetch(ptr BlockPointer, block Block,
```

Comment:
Function comment please (especially noting that the disk cache LRU time is updated, regardless of whether prefetching happened or not).